libbeat/reader/readfile/line.go

```diff
@@ -28,12 +28,15 @@ import (
 	"github.com/elastic/beats/v7/libbeat/logp"
 )
 
+const Unlimited = 0
```

Comment:
Please do not export this variable if you are not using it outside the `readfile` package.

Reply:
This constant potentially could be used for the public NewLineReader config param.

Reply:
Personally, I am not a huge fan of exposing something we would potentially use. In general, it is easier to export something unexported, then unexport something exported.

But I do not block this PR because of a philosophical question about a small constant.

Reply:
Sounds good, will update to not export.